### PR TITLE
Update docs 3.9.1 content

### PR DESCRIPTION
**Pull request by oscarpolanco**
# Step to test
- Go to the `website` directory
- Install the dependencies using `npm install`
- Use `npm start`
- Go to [localhost](http://localhost:3000/claycli/docs/cli.html)
- Compere your [local](http://localhost:3000/claycli/docs/cli.html) with the current [gh-page](https://clay.github.io/claycli/docs/cli.html)

### CODE REVIEW

File: docs/cli.md [hunk 1]
```diff
@@ -62,8 +62,8 @@ When running `claycli` programmatically (i.e., `import { someMethod } from 'clay
  * `warning` Shows a potentially undesired situation
  * `info` Gives the progress information of the current operation
  * `debug` Shows information of the current operation
- 
- As you can see, most of those correspond directly to log levels. Here you can check how to programmatically get the `claycli` methods on the [Programmatic API](#programmatic-api) section.
+
+ As you can see, most of those correspond directly to log levels.  On the [Programmatic API](#programmatic-api) section you can check how to programmatically get the `claycli` methods.
```
Comment (manuelurenah) [inline docs/cli.md]: Quick phrasing suggestion here
```suggestion
 As you can see, most of those correspond directly to log levels.  On the [Programmatic API](#programmatic-api) section you can check how to programmatically use the `claycli` methods.
```

File: docs/cli.md [hunk 2]
```diff
@@ -666,7 +666,7 @@ $ clay compile scripts --globs 'global/js/!(*.test).js'
 
 ### Project Specific Config File
 
-Not all projects are the same, and for project specific compilation changes you can add a `claycli.config.js` file to your project's root. This file must simply export an Object whose contains key/value pairs are read during compilation. Good use cases for this file include:
+Not all projects are the same, and for project specific compilation changes you can add a `claycli.config.js` file to your project's root. This file must simply export an Object that contains key/value pairs are read during compilation. Good use cases for this file include:
```
Comment (manuelurenah) [inline docs/cli.md]: Another phrasing suggestion
```suggestion
Not all projects are the same, and for project specific compilation changes you can add a `claycli.config.js` file to your project's root. This file must simply export an Object containing key/value pairs that are read during compilation. Good use cases for this file include:
```

File: website/versioned_docs/version-3.9.1/cli.md [hunk 3]
```diff
@@ -62,8 +62,8 @@ When running `claycli` programmatically (i.e., `import { someMethod } from 'clay
  * `warning` Shows a potentially undesired situation
  * `info` Gives the progress information of the current operation
  * `debug` Shows information of the current operation
- 
- As you can see, most of those correspond directly to log levels. Here you can check how to programmatically get the `claycli` methods on the [Programmatic API](#programmatic-api) section.
+
+ As you can see, most of those correspond directly to log levels.  On the [Programmatic API](#programmatic-api) section you can check how to programmatically get the `claycli` methods.
```
Comment (manuelurenah) [inline website/versioned_docs/version-3.9.1/cli.md]: Same suggestion as before https://github.com/clay/claycli/pull/131/files#diff-9bd77a22cd0e16427fed26b788d25194R66

File: website/versioned_docs/version-3.9.1/cli.md [hunk 4]
```diff
@@ -666,7 +666,7 @@ $ clay compile scripts --globs 'global/js/!(*.test).js'
 
 ### Project Specific Config File
 
-Not all projects are the same, and for project specific compilation changes you can add a `claycli.config.js` file to your project's root. This file must simply export an Object whose contains key/value pairs are read during compilation. Good use cases for this file include:
+Not all projects are the same, and for project specific compilation changes you can add a `claycli.config.js` file to your project's root. This file must simply export an Object that contains key/value pairs are read during compilation. Good use cases for this file include:
```
Comment (manuelurenah) [inline website/versioned_docs/version-3.9.1/cli.md]: Same suggestion as before https://github.com/clay/claycli/pull/131/files#diff-9bd77a22cd0e16427fed26b788d25194R669

**Comment by oscarpolanco** [comment]
PR updated. Thanks for the help @manuelurenah 😄

**Pull request merged**